### ISSUE TITLE
[Suggestions] Some tweaks and missing features

### ISSUE THREAD

**Issue by SatDog92** [paulevsGitch/BetterEnd#128]
I use Fabric v.1.11.1 on MC 1.16.5 and API v.0.30.3, and mod version is v.0.9.1-pre-1.16.4/5

Just a list of some features that could improve:
⬤ Despite not having Silk Touch, the other "leafy" leaves blocks are mined as themselves (already reported [here](https://github.com/paulevsGitch/BetterEnd/issues/126) but pointing it here too for the sake of completeness). Also, i think they should drop sticks too, and they drop way too many saplings (material for the composter, but still, i think they should drop sticks too).
⬤ There is an [already opened issue](https://github.com/paulevsGitch/BetterEnd/issues/125) about this: helix tree leaves cannot be mined with a hoe, and they cannot be composted either (as it happens with the other leaf types and vanilla wart blocks). Unlike with the other leaves, Silk Touch mechanics work well. Also, since they break like leaves, are they supposed to decay when no wood/solid block is around or they act like wart blocks?
⬤ Similarly to the Helix Tree Leaves, there are other blocks that cannot be mined with a hoe properly: Umbrella Tree Cluster, Glowing Pillar Luminophor, 
⬤ End Stone Dust cannot be mined quickly with shovel, and the Menger Sponges (both dry and wet) cannot be mined quickly with hoes like with vanilla sponges.
⬤ These Amber Land tall leaves from the ground... they're Lanceleaves, right? You have a seed for them but cannot acquire them on Survival, only on Creative, as these leaves drop nothing and don't even redirect to the item when middle-clicked. 
⬤ Sulfur Crystal cannot be mined by any means, not even with Silk Touch. Should they drop Sulfur for Gunpowder?
⬤ Lumecorn plant should drop more Lumecorn Seeds: not even with Fortune they can be multiplied, and this makes it really hard (if not impossible, because it's not a guaranteed drop, as far as i know) to make them any renewable. And i think that the Lumecorn Rods should work like Bamboo when broken, not like Nether vines. And when planted and not growth, they do not drop their seed as the other crops do. The same happens with Blue Vine Seeds with the Blue Vine Lantern plants and the Glowing Pillar Luminophor plant. I feel like they all should drop more than one seed when fully grown, similar to crops, because in this way they're barely renewable, the polar opposite of tree saplings (leaves do not drop sticks, as i wrote). The least we can have is to make seed drop be affected by Fortune.
⬤ Hydralux flowers do not have any kind of functionality. Saplings cannot be obtained in Survival and cannot grow with Bone Meal, despite being able to be planted. Any chance to have blocks and items related to it?
⬤ Are Mossy Glowshroom Caps supposed to drop saplings? I think they should act like the vanilla mushroom blocks, which drop mushrooms that are multiplied with Fortune, or themselves with Silk Touch, and that are mined faster either with an axe or a hoe (i prefer the axe in this case), i don't feel like instamining them with just hands feels right... and also have the same hardness of 
 mushroom blocks. Maybe i'm missing something here?
⬤ Some non-compostable things that should be composted:
 - Amber Grass;
 - Glowing Pillar Leaves;
 - Helix Tree Leaves (as pointed out before);
 - Glowing Pillar Luminophor;
 - Crystal Grass;
 - Shadow Plant;
 - Blooming Cooksonia;
 - Lumecorn Rod (i know it's weird but it's still a "corn", but i don't mind);
 - Vaiolush Fern;
 - Salteago;
 - Chorus Grass;
 - Cave Grass;
 - Bushy grass;
 - Tenanea Outer Leaves;
 - Glowing Bulb;
 - End Lily;
 - End Lily Leaf;
 - Blue Vine Fur;
 - Blue Vine Lantern;
 - Mossy Glowshroom Fur;
 - Mossy Glowshroom Hymenophore;
 - Mossy Gloshroom Cap;
 - Jungle Grass;
 - Umbrella Tree Cluster;
 - Umbrella Tree Membrane (ok, i know it's "jelly", but i thought it was still part of a mushroom, but not sure if this could be compostable);
 - Purple Jellyshroom Cap (same as above);
 - End Lotus Seed
 - Blossom Berry;
 - Hydralux Petal;
 - Hydralux Sapling;
 - Tube Worm;
 - all the berries;
 - Gelatine (should it?);
 - jelly food;
 - all the Charnia leaves.

Sorry for the long list, but i thought there should be more consistency with these items. I also tried to list these in base of what we have in Vanilla Minecraft. I might miss something, but that should be most of it.
EDIT: one more thing regarding worldgen: the caves (the ones with cave moss) are no more generated. Maybe they're just overly rare, but i remember finding them when i tried earlier versions. (and no, i didn't touch the biome config file, and inside there is the "generate caves" setting to true to most biomes).

**Comment by paulevsGitch** [comment]
Please don't make large single issue for many things together, split them into separate issues instead.

> Despite not having Silk Touch

This issue was already mentioned, please don't duplicate issues

> ... helix tree leaves cannot be mined with a hoe ...
> ... Similarly to the Helix Tree Leaves ...

The same as above

> End Stone Dust cannot be mined quickly with shovel

This bug is fixed in dev code

> Lanceleaves ... drop nothing ...

I will fix that

> Sulfur Crystal cannot be mined

Only the final stage of growing crystal can be mined

> Lumecorn plant should drop more Lumecorn Seeds

Will fix that

> Hydralux flowers do not have any kind of functionality

Wrong

> Saplings cannot be obtained in Survival and cannot grow with Bone Meal

They can. There was a bug with drop, but it was fixed

> Are Mossy Glowshroom Caps supposed to drop saplings

They don't drop saplings

> Some non-compostable things that should be composted

I will fix that, list of items is not very necessary as all these things will be done from the code with conditions

> the caves (the ones with cave moss) are no more generated

Caves are rare and their generation will be redone in the next release

In summary - please, don't write issues like that. It is hard to answer each paragraph and will be hard to manage issues later

**Comment by paulevsGitch** [comment]
As a result real issues list:
- [ ] Lanceleafs don't drop saplings
- [ ] End Stone Dust tool (check again)
- [ ] Lumecorn should drop at least one seed
- [ ] Add missing items to composter

**Comment by SatDog92** [comment]
> Please don't make large single issue for many things together, split them into separate issues instead.
Sorry, was afraid i would be "spamming" by doing that, but next time i will make separate threads instead.

**Comment by paulevsGitch** [comment]
It's much easier to manage separate issues and marked them as solved than managing single gigantic one

**Comment by paulevsGitch** [comment]
Fixed in 0.9.2
